src/app/template-list/template-list.component.ts

```diff
@@ -64,8 +64,18 @@ export class TemplateListComponent implements OnInit {
       this.router.navigate(['template',1])
     }
 
-    
-    
+    removeTemplate(tmpl): void {
+      this.templatesService.currTmpl = tmpl;
+      this.templatesService
+      .removeTemplate(tmpl).
+      then(() => {
+        this.tmplList = this.tmplList.filter(t => t !== tmpl);
+        if(this.currTmpl === tmpl) {
+          this.currTmpl = null;
+        }
+      });
+    }
+
```

Comment:
Then I believe we'll need to have an event listener or something to reload the tmplList so the deleted one disappears